[2023-10-26T10:38:24.178+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: read_db_and_save_local.read_db scheduled__2023-10-25T00:00:00+00:00 [queued]>
[2023-10-26T10:38:24.187+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: read_db_and_save_local.read_db scheduled__2023-10-25T00:00:00+00:00 [queued]>
[2023-10-26T10:38:24.188+0000] {taskinstance.py:1359} INFO - Starting attempt 9 of 9
[2023-10-26T10:38:24.200+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): read_db> on 2023-10-25 00:00:00+00:00
[2023-10-26T10:38:24.204+0000] {standard_task_runner.py:57} INFO - Started process 114 to run task
[2023-10-26T10:38:24.207+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'read_db_and_save_local', 'read_db', 'scheduled__2023-10-25T00:00:00+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/read_db_and_save_local.py', '--cfg-path', '/tmp/tmpmco627u1']
[2023-10-26T10:38:24.210+0000] {standard_task_runner.py:85} INFO - Job 23: Subtask read_db
[2023-10-26T10:38:24.251+0000] {task_command.py:415} INFO - Running <TaskInstance: read_db_and_save_local.read_db scheduled__2023-10-25T00:00:00+00:00 [running]> on host 88a410f69adf
[2023-10-26T10:38:24.438+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='read_db_and_save_local' AIRFLOW_CTX_TASK_ID='read_db' AIRFLOW_CTX_EXECUTION_DATE='2023-10-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='9' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-25T00:00:00+00:00'
[2023-10-26T10:38:24.446+0000] {base.py:73} INFO - Using connection ID 'mydb' for task execution.
[2023-10-26T10:38:24.454+0000] {logging_mixin.py:151} INFO - (('Danasawin', '19'),)
[2023-10-26T10:38:24.455+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-10-26T10:38:24.463+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=read_db_and_save_local, task_id=read_db, execution_date=20231025T000000, start_date=20231026T103824, end_date=20231026T103824
[2023-10-26T10:38:24.502+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-26T10:38:24.521+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
